src/components/card/card.component.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import './card.styles.css'
+
+import axios from 'axios'
+
+class Card extends Component {
+    state = {
+        
+        comments: [],
+        headings: ['Small Features', 'Booking Engine', 'Rates', 'CC Gateway', 'CRM/Email Marketing']
+
+    }
+
+    componentDidMount() {
+        axios.get('https://jsonplaceholder.typicode.com/comments?_limit=15')
+        .then(res => {
+        this.setState({comments:res.data}, console.log(this.state))
+       
+        })
+        .catch(error => 
+            {console.log(error)})
```

Comment:
No need for the braces here since it's an arrow function so it will directly return the `console.log`